src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,9 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
-        return false;
+        User user = UserService.findByEmail(email);
```

Comment:
```suggestion
        User user =  userService.findByEmail(email);
```

Reply:
It is a class level field, use it to create only one service object per Object, not for every method call